packages/url/src/index.js

```diff
@@ -9,7 +9,7 @@ import { createFilter } from 'rollup-pluginutils';
 
 const fsStatPromise = util.promisify(fs.stat);
 const fsReadFilePromise = util.promisify(fs.readFile);
-
+const { sep } = path;
```

Comment:
nit: I prefer using `path.sep` so I know its from the node package. but no strong opinions here

Reply:
let's address this when we do the TS conversion next month